cmd/ctr/commands/run/run_windows.go

```diff
@@ -163,6 +163,11 @@ func NewContainer(ctx context.Context, client *containerd.Client, cliContext *cl
 			if idType == "" {
 				return nil, errors.New("devices must have a non-empty IDType")
 			}
+			parts := strings.Split(devID, ":")
```

Comment:
As per the original issue in moby referred above, this can still cause issues when `devID` is `C:/foo`